migrations/10_remove_vote_quorum_migration.js

```diff
@@ -100,9 +91,16 @@ module.exports = function(deployer, network, accounts) {
       await memeRegistry.setFactory(memeFactoryInstance.address, true, Object.assign(opts, {gas: 100000}))
       await paramChangeRegistry.setFactory(paramChangeFactoryInstance.address, true, Object.assign(opts, {gas: 100000}))
 
-      writeSmartContracts(smart_contracts_path, smartContracts, env);
+    })
+    .then (async () => {
+
+      //  set last ran tx
+      const migrations = Migrations.at (migrationsAddress);
+      await migrations.setCompleted (10, Object.assign(opts, {gas: 100000}));
```

Comment:
maybe we should copy this step to every migration?, will not make a difference now but in case we decide to use a older migration as a template we don't forget this step